### PR TITLE
🐛 Bumping timeouts

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Base timeouts for getting the status and creating a service was 10 seconds instead of the original 20 seconds.
Restored them back to the previous intended threshold.

This was causing requests to timeout faster than expected in the webserver.

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/services/director_v2/_thin_client.py [hunk 1]
```diff
@@ -19,16 +19,18 @@
...
 
 class DirectorV2ThinClient(BaseThinClient, AttachLifespanMixin):
     def __init__(self, app: FastAPI) -> None:
         settings: ApplicationSettings = app.state.settings
-
         super().__init__(
-            request_timeout=10,
+            request_timeout=DEFAULT_LEGACY_TIMEOUT_S,
             base_url=settings.DYNAMIC_SCHEDULER_DIRECTOR_V2_SETTINGS.api_base_url,
             timeout=Timeout(10),
```
Comment (pcrespov) [inline services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/services/director_v2/_thin_client.py:35]: And this hardocded timeout of 10? What i... what is the different between request_timeout and timeout parameters?

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py [hunk 2]
```diff
@@ -17,7 +17,11 @@
 
 # To avoid any side effects using the same timeout as the
 # aiohttp client that was calling into director-v2 from webserver
-_DEFAULT_LEGACY_TIMEOUT_S: Final[NonNegativeInt] = 20
+DEFAULT_LEGACY_TIMEOUT_S: Final[NonNegativeInt] = 20
```
Comment (pcrespov) [inline packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py:20]: where is aiohptt constraint to 20?

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py [hunk 3]
```diff
@@ -17,7 +17,11 @@
...
+
+# make sure RPC calls time out after the HTTP requests
+# from dynamic-scheduler to director-v2 time out
+_RPC_DEFAULT_TIMEOUT_S: Final[NonNegativeInt] = 20 + 1
```
Comment (sanderegg) [inline packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py:24]: 1 second? is that really enough? I don't want to play the sleep time thing here. but usually times like these tend to smell bad... and have various constraints depending on the machines where they run etc etc... I thought the idea was to go with order of magnitudes, or maybe double the first one... well your call here.

**Review by sanderegg — APPROVED** [review]
all good. just maybe give a thought on my comment if possible. thanks!

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by pcrespov — APPROVED** [review]
thx

**Pull request merged**